### DISCUSSION TITLE
Phantom button touches with TuYa TM-YKQ004?

### DISCUSSION THREAD

**Discussion by deviantintegral** [Koenkk/zigbee2mqtt#22732] [Devices]
I set up a https://www.zigbee2mqtt.io/devices/TM-YKQ004.html as a light switch for one of my kid’s bedrooms. They love the physical construction of the remote and the glow-in-the-dark buttons.

However, I’ve noticed every few days that it sends a button push on buttons 3 or 4, turning on the light (based on my Home Assistant automations). At first, I thought that some curtains were touching and triggering it, but after moving it the buttons still trigger.

Has anyone else experienced this?

Here’s the AliExpress link: [Tuya ZigBee Wireless Touch Smart Switch 4 Gang 12 Scene Remote Control Light Smart Home Automation Works with Zigee Gateway ](https://www.aliexpress.com/item/1005004963401347.html?spm=a2g0o.order_list.order_list_main.11.35e21802ehjejz)

**Comment by deviantintegral** [comment]
I was able to sniff traffic from this device and confirmed that it's sending spurious button pushes. This isn't a problem in zigbee2mqtt, and I don't see any way based on the traffic sent to filter these out.

**Comment by deathrattle666** [comment]
Has anyone an idea how to reset this device? I removed the battery for several minutes and pressed several buttons afterwards. It stuck/does not re-pair again to zigbee2mqtt on HA.

THX!

**Reply by deviantintegral** [reply]
I’ve reset mine many times by pressing and holding the little button on the back of it for around 10 seconds. Perhaps you have the magnet mount on the back covering it up?

**Reply by deathrattle666** [reply]
Thank you Andrew for your reply.
Unfortunately there is no button on my device. Do you have the device model TM-YKQ004 or is it a variant?
Here is a picture of it: https://www.zigbee2mqtt.io/devices/TM-YKQ004.html#tuya-tm-ykq004

**Reply by deviantintegral** [reply]
Oh, oops I was mixed up in my devices. With this one, if you press and hold the top button for 10 seconds or so it will reset.

**Reply by deathrattle666** [reply]
Hi Andrew !
Sorry for answering delayed. I lost track on this a bit.

This is what I tried already. Unfortunately it did not work. None of long-pressing on any button was successfull. In every case I removed the battery for several minutes and tried pairing directly in the nearer area of the coordinator (sonoff dongel-e).

I removed the device from my zigbee2mqtt database/configuration and deleted every entry of it with the "mqtt-explorer" in the mqtt broker.

It will not pair again.

I start the pairing-mode of my zigbee coordinator and place the (new - 100% capacity) battery in, and then the device "boots" with the blue led on for approx. half a second.  After that I press one button (tried the procedure with all of them) and hold it. The blue led is on for round about three seconds. Then it goes dark. Still holding the button does not bring any further improvement.

It seems, that the device (meaning the remote control) is stucked in previous paired-mode. 

I am out of ideas on this and would be very grateful, if you maybe have another "suggestion"....

**Reply by deviantintegral** [reply]
Given mine had the issue with phantom touches, I would guess your buttons are broken but in a different way. Unfortunately, I don’t think this particular device was designed or constructed very well. I would probably just replace it with something else.

**Reply by deathrattle666** [reply]
Okay. Thanky for your feedback. If I ever find a solution, I will post it here.
